### ISSUE TITLE
Llevigar Boat Glitch

### ISSUE THREAD

**Issue by mepinoy27** [Wynncraft/Issues#7081]
So when we're on the ocean after giving the Sea Merchant a ticket, he teleports us to the ocean on the boat and simply stays there without actually reaching Llevigar. This is the second time it has happened and I'm not too sure this is how it's supposed to work. Any help or fix would be much appreciated :)

**Comment by LordHelmchen98** [comment]
You're right thats not how its supposed to work. Did you do it on different servers? Sometimes it doesnt work on specific server. Where did you start and can you try if it works from another city? And can you describe a bit more precisely what you did before/how you entered the boat?

**Comment by DebugingLonks** [comment]
works from me from nemract...

**Comment by DebugingLonks** [comment]
It seems they have completely removed the Sea Skipper's transportation. Everytime I  use it now I instantly get tped to the place I selected, no more dialogue. Or perhaps another bug?
